netbout-web/src/test/java/com/netbout/dynamo/DyAttachmentITCase.java

```diff
@@ -84,4 +85,57 @@ public void createsAndLoadsAttachment() throws Exception {
         );
     }
 
+    /**
+     * DyAttachment can obtain author of an attachment.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void obtainsAuthor() throws Exception {
+        final String alias = "charlie";
+        final Aliases aliases =
+            new DyBase().user(new URN("urn:test:89636")).aliases();
+        aliases.add(alias);
+        final Inbox inbox = aliases.iterate().iterator().next().inbox();
+        final Bout bout = inbox.bout(inbox.start());
+        final Attachments attachments = bout.attachments();
+        final String name = "name";
+        attachments.create(name);
+        final Attachment attachment = attachments.get(name);
+        MatcherAssert.assertThat(
+            attachment.author(), Matchers.equalTo(alias)
+        );
+        attachments.delete(name);
+    }
+
+    // @todo #839:30min DyAttachment Item should have an attribute for the
+    //  creation time of the attachment. Remove @Ignore from this test to check
```

Comment:
@mkordas Creation time field does not exist in remote db, so `DyAttachment.date()` crashes.